.github/actions/setup-httpbin-server/action.yml

```diff
@@ -70,6 +79,13 @@ runs:
         context: .
         cache-from: type=gha
         cache-to: type=gha,mode=max
+
+    - name: Load httpbin docker image
+      if: ${{ !env.ACT }}
+      uses: ./.github/actions/load-docker-image
+      with:
+        image: "kennethreitz/httpbin"
```

Comment:
Let's also do it for the nginx image like we said?

Reply:
I thought we had decided not to explicitly cache the nginx image since it will be implictly cached by `docker/build-push-action` on the `Build httpbin-proxy image` step.

Reply:
Ok. Well, is there a way that both the nginx and httpbin images be cached and loaded in the same way? Even if we have to create our own Dockerfile for the httpbin one. The goal I was trying to aim for was consistency in the loading/caching of all our Docker images used in CI.

Reply:
Ok no need, they're too different. Merged!